tests/sockshop/manifests/26-user-svc.yaml

```diff
@@ -5,14 +5,19 @@ metadata:
   name: user
   annotations:
         prometheus.io/scrape: 'true'
+        prometheus.io/port: '7070'
   labels:
     name: user
   namespace: sock-shop
 spec:
   ports:
     # the port that this service should serve on
   - port: 80
+    name: http
     targetPort: 80
+  - port: 7070
```

Comment:
is it ok that `user-dep` has no such `containerPort`?

Reply:
Not specifying a containerPort doesn't prevent that port from being exposed.
See [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#container-v1-core)